src/Symfony/Component/HttpKernel/Config/FileLocator.php

```diff
@@ -75,13 +75,13 @@ public function locate($file, $currentPath = null, $first = true)
                     }
```

Comment:
shouldn't it be a `continue` statement instead of `return $locations`?

Reply:
see https://github.com/symfony/symfony/pull/34886#discussion_r355962824 for the explanation I guess.